finetuner/__init__.py

```diff
@@ -485,3 +488,5 @@ def encode(
             inputs = model._move_to_device(inputs)
             output = model.run(inputs)
             batch.embeddings = output.detach().cpu().numpy()
+
+    return data
```

Comment:
Since the encode method previously worked on the input `DocumentArray` in place, I was not sure how to return the embeddings in the case that the user provided a list. Currently I am just returning the final DocumentArray in both cases, though I would welcome alternative suggestions